### PR TITLE
fixed card learning RTK

### PR DESCRIPTION
**Pull request by joycy-c**
# Description

Card Learning fully connected using RTK, should fix the progress bar bug and card burnt still showing bug

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

### Please add ticket link
https://trello.com/c/e4iPjcEI

### CODE REVIEW

File: .deployment/data/.mongodb/mongosh/64ce1227f95682852b8daf6e_log
Comment (wykswr) [inline .deployment/data/.mongodb/mongosh/64ce1227f95682852b8daf6e_log:1]: Remove all these local record files

File: client/.env
Comment (wykswr) [inline client/.env:1]: What's this new URL for?

File: client/src/redux/api/apiSlice.js
Comment (wykswr) [inline client/src/redux/api/apiSlice.js:1]: Didn't register "allCards" in the tagList, potential bugs here

File: .deployment/data/.mongodb/mongosh/64ce1227f95682852b8daf6e_log
Comment (joycy-c) [reply .deployment/data/.mongodb/mongosh/64ce1227f95682852b8daf6e_log:1]: local record files removed

**Pull request closed without merging**